### DISCUSSION TITLE
Use local MPV or add MPV options

### DISCUSSION THREAD

**Discussion by Bush-cat** [jellyflix-app/jellyflix#119] [Ideas]
I like having my mpv customizations, config, scripts and stuff

**Comment by jdk-21** [comment]
I just added supported for custom mpv.conf in Jellyflix ([c3f4a78](https://github.com/jellyflix-app/jellyflix/commit/c3f4a78218b84097473ad73abfc94a5a4855f4c8))
Since we use libmpv, not every setting will work thogh,
